### DISCUSSION TITLE
[Feature] Mass Email Action - Send to multiple email address

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#14189] [Feature improvement]
Hi,

How about sending to multiple email address ?
For example, in Contact module -> send email to primary + secondary email
It would be helpful when you have to send an urgent or very important message.

Regards,

**Comment by apcloic** [comment]
Hi all,

I've tried to do it by myself, I don't know if it's the right way but it works for me.
https://github.com/YetiForceCompany/YetiForceCRM/pull/13642

**Comment by bpabiszczak** [comment]
Why do you want to send the same email twice to one person?

**Comment by apcloic** [comment]
@bpabiszczak Because in case of important/urgent information, I need to maximize the chance that email was read.
